### PR TITLE
fix: svg icons in markdown for about us, update path strategy for pagination component, update redirects in source control 

### PR DESCRIPTION
**Pull request by evelynhathaway**

### CODE REVIEW

File: config/redirects_unicorn-utterances.com [hunk 1]
```diff
@@ -2,11 +2,11 @@
 # include this file with "include redirects_unicorn-utterances.com;"
 
 # url migration per PR #15
-location /authors/crutchcorn {
+location = /authors/crutchcorn/ {
     return 301 /unicorns/crutchcorn;
 }
 
-location /unicorns {
+location = /unicorns/ {
```
Comment (evelynhathaway) [inline config/redirects_unicorn-utterances.com:9]: These are the fixed versions that were deployed but never updated here

File: content/site/about-us.md [hunk 2]
```diff
@@ -23,31 +23,26 @@ We have various social media accounts where we share updates and new content wit
             <svg width="100" aria-hidden="true" viewBox="0 0 71.41206 50"><defs/><style>.st1{fill:var(--backgroundColor);} .st2{fill:var(--highImpactBlack);}</style><g transform="scale(.58824)"><path class="st2" fill-opacity="1" d="M118.9 13.3c-1.4-5.2-5.5-9.3-10.7-10.7C98.7 0 60.7 0 60.7 0s-38 0-47.5 2.5C8.1 3.9 3.9 8.1 2.5 13.3 0 22.8 0 42.5 0 42.5s0 19.8 2.5 29.2C3.9 76.9 8 81 13.2 82.4 22.8 85 60.7 85 60.7 85s38 0 47.5-2.5c5.2-1.4 9.3-5.5 10.7-10.7 2.5-9.5 2.5-29.2 2.5-29.2s.1-19.8-2.5-29.3z"/><path class="st1" d="M80.2 42.5L48.6 24.3v36.4z"/></g></svg>
         </a>
     </li>
-
```
Comment (evelynhathaway) [inline content/site/about-us.md:26]: All I needed to do to fix the SVG being rendered as markdown code was to remove the spacing

File: gatsby-node.js [hunk 3]
```diff
@@ -162,14 +162,14 @@ exports.createPages = ({ graphql, actions }) => {
...
-			path: `/`,
+			path: "/",
 			component: postList,
 			context: {
 				limitNumber: postsPerPage,
 				skipNumber: 0,
 				pageIndex: 1,
 				numberOfPages,
-				relativePath: ""
+				absolutePath: "/"
```
Comment (evelynhathaway) [inline gatsby-node.js:172]: the relative paths were not what the pagination component needed, so I updated them to absolute paths with slashes on both ends

File: src/components/series-toc/series-toc.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 /**
  * This is both a JS file and using CJS because it's used in `gatsby-config`
  */
-exports.SeriesToC = context => {
+exports.SeriesToC = (context) => {
```
Comment (evelynhathaway) [inline src/components/series-toc/series-toc.js:4]: Some of this is just code style Prettier was trying to make me do haha

File: src/templates/blog-profile/blog-profile.test.tsx [hunk 5]
```diff
@@ -92,11 +92,11 @@ test("Blog profile page renders", async () => {
 	).toBeInTheDocument();
 
 	fireEvent.click(await findByText("Post title"));
-	expect(onGarsbyLinkClick).toHaveBeenCalledTimes(2);
+	expect(onGatsbyLinkClick).toHaveBeenCalledTimes(2);
 
 	const authorImgs = await findAllByTestId("author-pic-0");
 	fireEvent.click(authorImgs[0]);
-	expect(onGarsbyLinkClick).toHaveBeenCalledTimes(4);
+	expect(onGatsbyLinkClick).toHaveBeenCalledTimes(4);
```
Comment (evelynhathaway) [inline src/templates/blog-profile/blog-profile.test.tsx:99]: Spelling

**Comment by evelynhathaway** [comment]
Deployed to https://fix-pagination-paths-svg-markdown.unicorn-utterances.com/

File: .github/workflows/deploy-dev.yml [hunk 6]
```diff
@@ -4,7 +4,7 @@ on: pull_request
 
 jobs:
   deploy:
-    if: github.repository_owner === 'unicorn-utterances'
+    if: github.repository_owner == 'unicorn-utterances'
```
[the remark this answers is not in the document]
Comment (crutchcorn) [reply .github/workflows/deploy-dev.yml:7]: Haha! Done that before!

File: content/site/about-us.md (hunk 2 above)
Comment (crutchcorn) [reply content/site/about-us.md:26]: LOL I hate that this file is this sensitive. My fault, but ugh

File: gatsby-node.js [hunk 7]
```diff
@@ -148,7 +148,7 @@ exports.createPages = ({ graphql, actions }) => {
 			}
 
 			createPage({
-				path: `posts${post.node.fields.slug}`,
+				path: `/posts${post.node.fields.slug}`,
```
[the remark this answers is not in the document]
Comment (crutchcorn) [reply gatsby-node.js:151]: Huh! This worked in Gatsby 2, must've changed with 3

File: src/templates/blog-profile/blog-profile.test.tsx (hunk 5 above)
Comment (crutchcorn) [reply src/templates/blog-profile/blog-profile.test.tsx:99]: LOL Garsby

**Review by crutchcorn — APPROVED** [review]
This all looks fantastic, thank you so much for helping getting this working

**Pull request merged**